config.md

```diff
@@ -32,7 +32,7 @@ For example, if a configuration is compliant with version 1.1 of this specificat
   The path is either an absolute path or a relative path to the bundle.
   On Linux, for example, with a bundle at `/to/bundle` and a root filesystem at `/to/bundle/rootfs`, the `path` value can be either `/to/bundle/rootfs` or `rootfs`.
   A directory MUST exist at the path declared by the field.
-* **`readonly`** (bool, OPTIONAL) If true then the root filesystem MUST be read-only inside the container, defaults to false.
+* **`readonly`** (bool, OPTIONAL) If true then the root filesystem MUST be read-only inside the container, defaults to false. On Windows, this field must be omitted or false.
```

Comment:
Previous discussion in #23 and #476.

An alternative to this would be to label the property as Linux/Solaris-specific, so it wouldn't be specified for Windows and Windows runtimes could silently ignore it (using [this][1]).  That gets us to the runtime behavior proposed in #476, but avoids confusing config authors (like the spec wording in #476) because Windows authors would know that the property did not apply to their platform, and non-Windows authors would know that the property did apply (and runtimes would have to implement it) on their platforms.

[1]: https://github.com/opencontainers/runtime-spec/blob/559acdf05cd4dd0b57d720a3981147597d249280/config.md#extensibility